R/stopifnotproject.R

```diff
@@ -52,7 +52,9 @@
 
     message(
       paste0(c(paste0(directory, basename(path),
-                      " is not a ProjectTemplate directory"),
+                      " is not a ProjectTemplate directory.  ", 
```

Comment:
Perhaps change it to
```suggestion
                      " is not a valid ProjectTemplate directory because one or more mandatory directories are missing.  ", 
```